docs/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: home
+
+# Header Content
+# =================================
+header:
+    - logo_image: assets/img/dsa_logo.svg
+
+# Banner Content
+# =================================
+title: Harness the full potential of your digital pathology data
+subtitle: A containerized web-based platform for the analysis, visualization, management and annotation of whole-slide digital pathology imaging data
+
+hero_image: assets/img/dsa_hero.jpg
+
+hero_buttons:
+  - link_text: View Demo
+    link: #
+
+  - link_text: Read Docs
+    link: #
+
+  - link_text: View on GitHub
+    link: #
+
+# What is DSA Section
+# =================================
+about_title: What is DSA?
+about_text: The Digital Slide Archive (DSA) is a platform  that provides the ability to store, manage, visualize and annotate large imaging data sets.  The DSA is an umbrella term and consists of an analysis toolkit (HistomicsTK), a library to manage/visualize slides (HistomicsUI) as well as a database layer (using Mongo), and a web-server that provides a rich API.  This system allows;
```

Comment:
I'd reword this somewhat and not end on a semicolon.  Perhaps: `The Digital Slide Archive (DSA) is a platform that provides the ability to store, manage, visualize and annotate large imaging data sets.  The DSA consists of an analysis toolkit (HistomicsTK), an interface to visualize slides and manage annotations (HistomicsUI), a database layer (using Mongo), and a web-server that provides a rich API and data management tools (using Girder).  This system can:`

Reply:
Fixed in 27f408b8d0311cdf44401bbdd023452f144ef19f

Reply:
Fixed in 3ca419c83a963bdafaf7e336b4cda927c524722b